test-cases/features/ics_space_amplification_test.yaml

```diff
@@ -0,0 +1,21 @@
+test_duration: 300
+
+round_robin: true
+
+n_db_nodes: 3
+n_loaders: 1
+n_monitor_nodes: 1
+
+instance_type_db: 'i3.4xlarge'
+instance_type_loader: 'c4.large'
+
+nemesis_class_name: 'NoOpMonkey'
+user_prefix: 'ics_space_amplification'
+server_encrypt: false
+client_encrypt: false
+
+
+use_mgmt: false
+nemesis_during_prepare: false
+
+append_scylla_args: '--compaction-enforce-min-threshold 1'
```

Comment:
since this overwrite the defaults we have in `test_defaults.yaml`, double check if you don't want to include that too

Reply:
that's may be right. i can't tell yet since i need a clarification about this argument from @raphaelsc

Reply:
I've meant the other argument we put as default. You might want them too, right now only this argument would be used.

Reply:
yaron, I think it's not good to have that flag turned on by default. Users will likely not use it. So better test ICS with defaults, no? We have plans to improve that, but by now, I think we should leave that flag turned off as users will likely do so too.

Reply:
@raphaelsc , that make sense. the consequence will be the test fails on first step. so do we consider that a real issue that should be handled in a github issue?

Reply:
in any case, i opened task: https://trello.com/c/6LpWsBON/1437-turn-off-appendscyllaargs-compaction-enforce-min-threshold-1 .
so this PR could be meanwhile concluded and merged.

Reply:
@yarongilor you are still missing my comment here, that the other commandline argument we add as default in out tests are overwritten by this line.
please make that is the intended effect you want,

look at the defaults, when you set this in your yaml, only what configured in you yaml for this value is used
https://github.com/scylladb/scylla-cluster-tests/blob/9ddb0e6f6b4fabf4ae7eaf1c3029cf51fcc1a0da/defaults/test_default.yaml#L13

Reply:
> add as default in out tests are overwritten by this line.
> please make that is the intended effect you want,
> 
> look at the defaults, when you set this in your yaml, only what configured in you yaml for this value is used

i guess there is something misleading here with the naming, since this is, of course, not intended to override other default, but only to add one. so the name "append_scylla_args" causes a confusion. it doesn't append but rather overrides the default. i would suggest to change it so that it will actually append and given value to the default values list as long as it's not contained in default list already. perhaps for an override option we can add a new parameter named "set_scylla_args".

Reply:
for this PR, the flag is now removed: 
06c3c14f - (HEAD -> ics_space_amplification, origin/ics_space_amplification) Removed append_scylla_args: '--compaction-enforce-min-threshold 1